package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sentry-cli",
+  "version": "0.25.0",
+  "description": "A command line utility to work with Sentry. https://docs.sentry.io/hosted/learn/cli/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "preinstall": "INSTALL_DIR=./bin ./bin/install-sentry-cli"
```

Comment:
Let me know which path you prefer to take and i can modify/close the PR as needed